result/src/test/kotlin/com/github/kittinunf/result/FactoryTests.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.kittinunf.result
+
+import org.hamcrest.CoreMatchers
+import org.hamcrest.CoreMatchers.equalTo
+import org.hamcrest.CoreMatchers.instanceOf
+import org.junit.Test
+import org.hamcrest.MatcherAssert.assertThat
+
+class FactoryTests {
+    @Test
+    fun testRunCatching() {
```

Comment:
It looks like this test `inline fun <V> runCatching(block: () -> V)` , could you please add another one for another version of `runCatching`?

Reply:
Done.